pkg/util/openstack/keymanager.go

```diff
@@ -22,7 +22,7 @@ import (
 
 	"github.com/gophercloud/gophercloud"
 	"github.com/gophercloud/gophercloud/openstack/keymanager/v1/secrets"
-	"k8s.io/cloud-provider-openstack/pkg/cloudprovider/providers/openstack/metrics"
+	request_metrics "k8s.io/cloud-provider-openstack/pkg/metrics"
```

Comment:
Can `"k8s.io/cloud-provider-openstack/pkg/metrics"` be imported without giving it an alias - similar to `pkg/util/openstack/loadbalancer.go` and `pkg/util/openstack/network.go`?

Reply:
@seanschneeweiss just updated, thanks~